src/itables/datatables_format.py

```diff
@@ -83,15 +79,18 @@ def datatables_rows(df, count=None, warn_on_unexpected_types=False, pure_json=Fa
     """Format the values in the table and return the data, row by row, as requested by DataTables"""
     # We iterate over columns using an index rather than the column name
     # to avoid an issue in case of duplicated column names #89
-    if count is None or len(df.columns) == count:
-        empty_columns = []
-    else:
-        # When the header requires more columns (#141), we append empty columns on the left
-        missing_columns = count - len(df.columns)
-        assert missing_columns > 0
-        empty_columns = [[None] * len(df)] * missing_columns
 
-    try:
+    # TODO: why not if isinstance(df, pd.DataFrame)
+    if is_pandas_dataframe(df):
```

Comment:
`is_pandas_dataframe` checks if `df` is a pandas dataframe without importing pandas

This is useful if your package doesn't have pandas as a required dependency, and users might not have imported pandas already. pandas is a huge library so just importing it takes > half a second
```
$ time python -c 'import pandas'

real    0m0.662s
user    0m1.929s
sys     0m0.032s
```

However, if you're doing `import pandas as pd` at the top of the file anyway, then there's no benefit to `is_pandas_dataframe`, you may as well do the isinstance check with `pd.DataFrame`

Reply:
I see, thanks for explaining! On the long term I agree it would make sense to not import Pandas by default, I will think of that!